nebari/upgrade.py

```diff
@@ -23,14 +23,14 @@ def do_upgrade(config_filename, attempt_fixes=False):
     try:
         verify(config)
         rich.print(
-            f"Your config file [purple]{config_filename}[/purple] appears to be already up-to-date for qhub version [green]{__version__}[/green]"
+            f"Your config file [purple]{config_filename}[/purple] appears to be already up-to-date for Nebari version [green]{__version__}[/green]"
         )
         return
     except (ValidationError, ValueError) as e:
         if is_version_accepted(config.get("qhub_version", "")):
             # There is an unrelated validation problem
             print(
-                f"Your config file {config_filename} appears to be already up-to-date for qhub version {__version__} but there is another validation error.\n"
+                f"Your config file {config_filename} appears to be already up-to-date for Nebari version {__version__} but there is another validation error.\n"
```

Comment:
```suggestion
                f"Your config file {config_filename} appears to be already up-to-date for nebari version {__version__} but there is another validation error.\n"
```

Reply:
Hi @trallard, everything under the `upgrade,py` will be handled after the migration occurs, the reason is that we will need to rebuild part of that code to work with the different:
- `qhub-config.yaml` and `nebari-config.yaml`
- version schemas

The upgrade command `pytest` suit works by upgrading a qhub-config.yaml file from 0.3.* up until the currently installed version, and that will break if we change the names only (not only that, but the whole purpose will not make sense anymore)

I've created [this issue](https://github.com/Quansight/qhub/issues/1512) to go over the subject